README.md

```diff
@@ -1705,6 +1691,24 @@ Network mask length of the secondary IP address on the interface. Valid values a
 ###### `ipv4_arp_timeout`
 Address Resolution Protocol (ARP) timeout value. Valid values are integer and keyword 'default'. Currently only supported on vlan interfaces. This property is not supported on IOS XR as IOS XR does not support vlan interfaces.
 
+###### `ipv4_forwarding`
+IP forwarding state.  Valid values are string or keyword 'default'.
```

Comment:
"This property is not supported on IOS XR". Also please update the caveat table for this provider to include this same limitation.